test/unittests/mongoBackend/mongoUpdateContext_withOnchangeSubscriptions_test.cpp

```diff
@@ -3087,13 +2948,11 @@ TEST(mongoUpdateContext_withOnchangeSubscriptions, DISABLED_MongoDbQueryFail)
     BSONObj sub = connection->findOne(SUBSCRIBECONTEXT_COLL, BSON("_id" << OID("51307b66f481db11bf860001")));
     EXPECT_EQ(20000000, sub.getIntField("lastNotification"));
 
-    /* Release connection */
-    setMongoConnectionForUnitTest(NULL);
-
+    /* Restore real DB connection */
+    setMongoConnectionForUnitTest(connectionDb);
```

Comment:
ok.

I would like to put all these things in utInit/Exit (and use the functions in all tests).
utInit would have parameters (bool  setMongoConnection) etc to 'change' its behavior, to fit all needs, but implemented only in *one* place.

NTC